README.md

```diff
@@ -2,22 +2,131 @@
 
 # r7insight_docker
 
+- [r7insight_docker](#r7insight_docker)
+  - [Arguments](#arguments)
+    - [Mandatory arguments](#mandatory-arguments)
+      - [Token](#token)
+      - [Region](#region)
+    - [Optional arguments](#optional-arguments)
+      - [Filtering](#filtering)
+        - [Notes](#notes)
+        - [Filters](#filters)
+  - [Usage as a Container](#usage-as-a-container)
+    - [Docker Arguments](#docker-arguments)
+    - [Examples](#examples)
+    - [Running in a restricted environment](#running-in-a-restricted-environment)
+  - [Usage as a CLI](#usage-as-a-cli)
+  - [Embedded usage](#embedded-usage)
+  - [Building](#building)
+    - [Docker](#docker)
+      - [Using the Docker file](#using-the-docker-file)
+    - [Make](#make)
+    - [Node](#node)
+  - [How it works](#how-it-works)
+
 Forward all your logs to the [Rapid7 Insight Platform](https://www.rapid7.com/products/insight-platform/), like a breeze.
 
 ![InsightOps dashboard](https://raw.githubusercontent.com/rapid7/r7insight_docker/master/dashboard.png)
 
-You can download the community pack created by InsightOps at [Docker pack](https://insightops.help.rapid7.com/docs/docker). The community pack comes with pre-defined out-of-the-box alerts and widgets to get you started.
+## Arguments
+
+### Mandatory arguments
+
+#### Token
+
+You can supply the tokens using command line arguments:
+- `--logstoken`: Log token for logs
+- `--statstoken`: Log token for stats
+- `--eventstoken`: Log token for events
+- `--token`: Log token which is used for the above tokens if one is not provided.
+  * You can only supply this token and it'll be used for logs, stats and events. 
+
+You can also supply log, stats and event tokens using environment variables:
```

Comment:
Mention which one has priority?

Reply:
What do you mean by priority?

Reply:
If these are set but you also supply the explicit arguments.